packages/roosterjs-editor-core/lib/coreApi/insertNode.ts

```diff
@@ -179,3 +179,13 @@ export const insertNode: InsertNode = (core: EditorCore, node: Node, option: Ins
 
     return true;
 };
+function adjustInsertPositionNewLine(blockElement: BlockElement, core: EditorCore, pos: Position) {
+    let tempPos = new Position(blockElement.getEndNode(), PositionType.After);
+    if (safeInstanceOf(tempPos.node, 'HTMLTableRowElement')) {
+        const div = core.contentDiv.ownerDocument.createElement('div');
+        const range = createRange(pos);
+        range.insertNode(div);
```

Comment:
Wait, is this going to insert a DIV after a TR?

Can you give me an example including:
- HTML before insert
- The position
- HTML after insert

Reply:
No, it is going to insert a Div inside of a TD. To treat it as a new Line inside of a table cell

Reply:
Before Insert 
![image](https://user-images.githubusercontent.com/8291124/181527077-945f977b-fa8c-46ef-a9b5-b5a8d820156a.png)

Position
![image](https://user-images.githubusercontent.com/8291124/181527407-6a9747aa-aabc-4ef6-94f3-6c804865f31d.png)

After Insert
![image](https://user-images.githubusercontent.com/8291124/181527972-d67d1df6-aa8b-4e30-b16f-cacc80ab51da.png)